Dockerfile

```diff
@@ -67,8 +67,8 @@ RUN <<EOT
     find /usr/lib/python3.11 -type f -name '*.py[co]' -delete -o -type d -name __pycache__ -delete
 
     # Remove default configs because we will install our own
-    rm -f /etc/php/php.ini \
-        /etc/php/php-fpm.d/www.conf \
+    rm -f /etc/php81/php.ini \
+        /etc/php81/php-fpm.d/www.conf \
```

Comment:
@chrislrobinson the symlink is still there; this `rm` call just happens before the symlink (that we place) is made -- so this was a bugfix rather than a change. The link didn't exist yet, so the rm calls here were noop.

Reply:
@chrislrobinson thanks for heads-up on the BC break - have fixed that. I want to add tests for all the OPC/bc stuff, so hopefully we can have a bit more confidence going forward.